src/App.tsx

```diff
@@ -15,48 +16,136 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  visibleGoods.sort((goodFirst: string, goodSecond: string): number => {
```

Comment:
types for this callback are not required :)

```suggestion
  visibleGoods.sort((goodFirst, goodSecond) => {
```